### DISCUSSION TITLE
Get Tweets of Users with Appwrite

### DISCUSSION THREAD

**Discussion by zionnite** [appwrite/appwrite#6044] [General]
Currently working with Appwrite, and I'm trying to simulate my app to Twitter App where users can follow and unfollow each other.

At the moment, users can create tweet, like tweet, comment on tweet, follow and unfollow each other but I want to be able to fetched tweet of users I'm following, I don't want to pull all the tweets to the user, only the person they are following.

**Comment by PratikDev** [comment]
You need to work with the Twitter API to implement your idea. Appwrite itself doesn't have any connection with it IMO. Or maybe you need to describe your scenario more, if you found any issues, or want to request a new feature in Appwrite services

**Comment by joeyouss** [comment]
Hi @zionnite , I think issue is better suited for discussions. Moving it there for now but yes if you want a feature request, then sure you can open one!
